css-positioning/relative.css

```diff
@@ -0,0 +1,37 @@
+.pokemon {
+  width: 900px;
+  margin: 0 auto;
+  text-align: center;
+}
+
+.pokemon > img {
+  width: 150px;
+
+  /* your code here */
+  position: relative;
+}
+
+#bulbasaur {
+  /* your code here */
+  bottom: 25px;
+}
+
+#charmander {
+  /* your code here */
+  top: 100px;
+}
+
+#squirtle {
+  /* your code here */
+  top: 125px;
+}
+
+#pikachu {
+  /* your code here */
+  top: 55px;
+}
+
+#jigglypuff {
+  /* your code here */
+  top: 250px;
+}
```

Comment:
Your output doesn't match mine for this one. You have the right idea for the code, but the measurements are off from what would be needed for the example.